### PR TITLE
Menu settings addons

### PR DESCRIPTION
**Pull request by stefan-hoehn**
Documentation of the settings -> addons page

### CODE REVIEW

**Comment by Confectrician** [comment]
Looks like these changes are mixed up with the one from #1911, may you have alook over it before i start reviewing?

File: settings/automations.md [hunk 1]
```diff
@@ -5,7 +5,10 @@ title:  Automation
 
 # Settings - Automation
 
-This is the section
+This is the section to automate openHAB which provides rules, script and a scheduling sectiom.
+See [What's the Difference Between a Rule, Script, and Schedule?](/docs/tutorial/rules_introduction.html#what-s-the-difference-between-a-rule-script-and-schedule)
+
+
 
 {::options toc_levels="2..4"/}
```
Comment (Confectrician) [inline settings/automations.md:14]:
```suggestion

```

File: settings/automations.md [hunk 2]
```diff
@@ -14,12 +17,78 @@ This is the section
...
+A rule that is schedule repeatedly like everyday at 8:00 in the morning will therefore be shown on every day of the calendar view.
+The way openHAB handles this, is that a rule will get a tag "schedule", so if you create a rule with the help of the ![add scheduke](images/plus.png) at the bottom right corner, openHAB autoamtically adds that tag (that is also true if you create a rule with a time trigger via the normal rule page).
+In case you have timed that trigger that repeats very often and which would "pollute" the schedule view with far too many entries, you can just delete the tag "schedule" in the edit mode of that rule (it can always be added later again).
+
+The page has a searchable list which allows to filter by the name of the rule.
+
+To see more entries scroll to the left or the right.
+Note that the selection functionality is currently not supported.
+
+
```
Comment (Confectrician) [inline settings/automations.md:94]:
```suggestion
```

File: settings/index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+![settings-menu](images/settings-menu.png)
+
+The _Settings_ entry links to an overview page that itself has several subsections that lead you to the respective configuration pages..
+
+![img.png](images/settings-page.png)
+
+Some of these subpages, which are most often used, can also be accessed directly via the menu, e.g. either click on _Settings->Items_ in the overview page or on directly _Items_ in the menu.
+
+{::options toc_levels="2..4"/}
```
Comment (Confectrician) [inline settings/index.md:18]:
```suggestion
```

**Comment by stefan-hoehn** [comment]
As discussed offline it seems I did this branch from the last branch. Can you please try to merge anyway. I am going to be careful next time when I create a new branch that I make sure I am on main.

**Review by Confectrician — APPROVED** [review]
I have added a Sidebar entry for the index page and resolved all conflicts based on the wrong base branch. 🙂

**Pull request merged**